proto/substrait/algebra.proto

```diff
@@ -589,6 +591,13 @@ message WriteRel {
     WRITE_OP_CTAS = 4;
   }
 
+  enum CreateMode {
+    CREATE_MODE_UNSPECIFIED = 0;
+    CREATE_MODE_ERROR_IF_EXISTS = 1; // Throw an error if the table already exists (default behavior)
+    CREATE_MODE_REPLACE_IF_EXISTS = 2; // Replace the table if it already exists ("OR REPLACE")
+    CREATE_MODE_IGNORE_IF_EXISTS = 3; // Ignore the request if the table already exists ("IF NOT EXISTS")
+  }
```

Comment:
I don't know if it applies here but Spark also has an append mode.

https://spark.apache.org/docs/latest/api/python/reference/pyspark.sql/api/pyspark.sql.DataFrameWriter.mode.html

Reply:
Yeah, I guess append makes sense (although it isn't something that commonly exists in dbs)

Reply:
Added append.